content/docs/learning-guide/tutorials/remote-repositories/create-remote-repository.md

```diff
@@ -0,0 +1,193 @@
+<ul> 
+For example: Add your gem to your project when working locally:
+<ul>
+
+```
+o3de.bat edit-repo-properties --repo-path "C:\o3de_Projects\<RemoteProjectName>\repo.json" --add-gem "C:\o3de_Projects\<RemoteProjectName>\Gems\MyGem"
+```
+
+</ul>
+</ul>
+<br>
+
+However, pushing your changes to your version control repository would not allow other user to use your gem, move on to the next step if you want to create a downloadable `release archive` for your gem to be shared with the world. Make sure  you push all your changes before moving to the next step to share your gem with the world.
+</ul>
+</ul>
+
+<br>
+
+## Create Release Archive
+---
+The primary purpose of a **Release archive** is to provide a well-defined and stable version of the software that can be easily shared, installed, and distributed.<br> 
+
+In O3DE we provide the option for users to create a release archive with .zip format.<br>
+
+Due to user safty, we use `sha-256` to hash our files before creating the release archive. Thus, you won't be able to modify your `repo.json` file once your archive is created. Every time you make a change to your gem, including the repo.json file, the `sha-256` hash will change.<br>
+
+<ul>
+
+Note: You must know where your zip will be created online **download path** before you actually create the release archive zip.</ul>
+<br>
+<ul>
+
+### Add A Gem To Your Remote Repository And Create A Release Archive
+---
+Here is a Github example of how to create a release archive. <br><br>
+Note: `--release-archive-path` command can only be used after you call the `--add-gem` command.<br> See below example:
+
+<ul>
+
+<li>
+
+To create a release archive `download prefix`, make sure you follow this pattern for any Github repositories:<br>
+<ul>
+
+ `https://github.com/YourGitAccount/RemoteProjectName/releases/download/YouReleaseTag`<br>
+
+```
+o3de.bat edit-repo-properties --repo-path "C:\o3de_Projects\<RemoteProjectName>\repo.json" --add-gem "C:\o3de_Projects\<RemoteProjectName>\Gems\MyGem" --release-archive-path "C:\o3de_Projects\<RemoteProjectName>\Gems" --download-prefix "https://github.com/<YourGitAccount>/<RemoteProjectName>/releases/download/<YouReleaseTag>"
+```
+<ul>
+
+ Provide the following required information:
+ <li> --repo-path: the absolute path to your remote project's repo.json file
+ <li> --add-gem: the path to the directory to where the actual gem you want to add is located
+ <li> --release-archive-path: the path to where you want to place your .zip archive, it should be placed in the Gems folder of your remote project folder
+ <li> --download-prefix: the URI where your gem archive can be download from
+
+<br>
+1. Here is an example of where you should upload your release to Github (you can find this on your version control url):
+
+
+![tag](Media/add_release.png)
```

Comment:
Use the information in the following page to learn how to add images: https://www.docs.o3de.org/docs/contributing/to-docs/style-guide/media/